README.md

```diff
@@ -39,33 +39,73 @@ developer communities.
 * Built-in support for Imitation Learning
 * Flexible agent control with On Demand Decision Making
 * Visualizing network outputs within the environment
-* Simplified set-up with Docker
 * Wrap learning environments as a gym
 * Utilizes the Unity Inference Engine
 * Train using concurrent Unity environment instances
 
-## Documentation
+## Releases & Documentation
+**Our latest, stable release is 0.14.1. Click
+[here](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Readme.md) to
+get started with the latest release of ML-Agents.**
+
+The table below lists all our releases, including our `master` branch which is under active
+development and may be unstable. A few helpful guidelines:
```

Comment:
probably suffices to say "under active development" instead of "under active development and may be unstable"

Reply:
I want to emphasize the instability - as active development just sounds like they'll get the freshest version.